api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -270,12 +270,16 @@ message Permission {
     // Extension for configuring custom matchers for RBAC.
     // [#extension-category: envoy.rbac.matchers]
     core.v3.TypedExtensionConfig matcher = 12;
+
+    // Glob URL path matching.
+    // [#extension-category: envoy.rbac.matchers]
+    core.v3.TypedExtensionConfig glob_path = 13;
```

Comment:
I don't think i like naming this `glob_path` given that it possible covers more features than just globbing, that's why it's actually called `uri_template` 🤔

Reply:
I will replace `glob` everywhere with `uri_template`.